card/mod/basic_formats/set/all/base.rb

```diff
@@ -88,7 +88,9 @@ def date_view date
     end
   end
 
-  view :labeled_content, :core
+  view :labeled_content do
+    render_core
```

Comment:
difference:
the alias links all sets' labeled_content view to one specific core view.
render_core means the labeled_content view will vary if the core view varies.

(this was breaking authors on tickets)

Reply:
Doesn't `titled_content` have the same problem?